lib/showoff.rb

```diff
@@ -523,6 +525,25 @@ def process_content_for_section_tags(content, name = nil, opts = {})
       doc.to_html
     end
 
+    # TODO: damn, this one is bad. This moves the notes div outside of the .content div, then removes
```

Comment:
Maybe we shouldn't name it generically so it doesn't become a dumping ground for stuff like this?